src/sentry_freight/plugin.py

```diff
@@ -53,6 +53,12 @@ class FreightPlugin(ReleaseTrackingPlugin):
     description = 'Integrate Freight release tracking.'
     version = sentry_freight.VERSION
 
+    def has_project_conf(self):
```

Comment:
I updated the plugins endpoint to not require these updates to show freight in the release tracking view: https://github.com/getsentry/sentry/pull/5403/files#diff-afd6044965ad3dff0908c1c857909b77R16